### PR TITLE
Add triage and history go links

### PR DESCRIPTION
**Pull request by fejta**

### CODE REVIEW

**Comment by fejta** [comment]
@ixdy @rmmh it is really obnoxious that I cannot assign this to anyone

File: k8s.io/configmap-nginx.yaml [hunk 1]
```diff
@@ -187,9 +187,11 @@ data:
         location / {
           rewrite ^/bounty$          https://github.com/kubernetes/kubernetes.github.io/issues?q=is%3Aopen+is%3Aissue+label%3ABounty redirect;
           rewrite ^/help-wanted$     https://github.com/kubernetes/kubernetes/labels/help-wanted redirect;
+          rewrite ^/history$         https://storage.googleapis.com/kubernetes-test-history/static/index.html
```
Comment (ixdy) [inline k8s.io/configmap-nginx.yaml]: test-history might be a better shortname?

**Comment by ixdy** [comment]
Why did you close this?

**Comment by fejta** [comment]
Lame, I can't reopen it?

File: k8s.io/configmap-nginx.yaml (hunk 1 above)
Comment (fejta) [inline k8s.io/configmap-nginx.yaml]: Done

**Comment by ixdy** [comment]
You also appear to have deleted the branch, which means I can't reopen it either.

**Comment by fejta** [comment]
ha!

**Pull request closed without merging**